README.md

```diff
@@ -121,30 +121,36 @@ This example defines 3 routes:
 // src/App.js
 
 import React, { Component } from 'react';
-import { BrowserRouter as Router, Route } from 'react-router-dom';
-import { SecureRoute, Security, LoginCallback } from '@okta/okta-react';
+import { BrowserRouter as Router, Route, withRouter } from 'react-router-dom';
+import { Security, LoginCallback, useOktaAuth } from '@okta/okta-react';
+import _SecureRoute from '@okta/okta-react/src/SecureRoute';
```

Comment:
The `_` prefix normally means something private, but here we're just using it to mean something react-router specific.

Could we (1) place SecureRoute into something that makes it clear it is react-router specific rather than just `src/`?  and (2) remove the `_` prefix?